R/functions-eager.R

```diff
@@ -47,21 +50,15 @@
 #' df2 <- pl$DataFrame(a = 2L, c = 4L)
 #' pl$concat(df1, df2, how = "diagonal")
 #'
-# TODO-REWRITE: uncomment this
-# df_a1 <- pl$DataFrame(id = 1:2, x = 3:4)
-# df_a2 <- pl$DataFrame(id = 2:3, y = 5:6)
-# df_a3 <- pl$DataFrame(id = c(1L, 3L), z = 7:8)
-# pl$concat(df_a1, df_a2, df_a3, how = "align")
+#' df_a1 <- pl$DataFrame(id = 1:2, x = 3:4)
+#' df_a2 <- pl$DataFrame(id = 2:3, y = 5:6)
+#' df_a3 <- pl$DataFrame(id = c(1L, 3L), z = 7:8)
+#' pl$concat(df_a1, df_a2, df_a3, how = "align")
+#' pl$concat(df_a1, df_a2, df_a3, how = "align_left")
+#' pl$concat(df_a1, df_a2, df_a3, how = "align_right")
 pl__concat <- function(
   ...,
-  how = c(
-    "vertical",
-    "vertical_relaxed",
-    "diagonal",
-    "diagonal_relaxed",
-    "horizontal",
-    "align"
-  ),
+  how = "vertical",
```

Comment:
When there are too many options (here 9 with many characters), it clutters the docs. All options are listed in the description of `how` so I don't think it's necessary to take so much space in the docs with this.

Reply:
How about list some of them? (Maybe only non-relaxed ones)
Like `tibble::as_tibble()`'s `.name_repair` argument.

https://github.com/eitsupi/neo-r-polars/blob/776e95621042b7f51947da2544db661d9b34cbab/man/s3-as_tibble.Rd#L11

Reply:
I don't like that, it would be misleading for the user IMO. If I go to the docs and see a list of accepted values for an argument in the "Usage" section then I assume that's all this argument can take.

Reply:
It is quite a pain for me to have to look at the help page to check the available values, but well, it is checked in error, so this is still not too bad.